### ISSUE TITLE
False positive - Rule XMLFILES Folder not found: media/css

### ISSUE THREAD

**Issue by anibalsanchez** [joomla-extensions/jedchecker#131]
Hi,

The extension has this declaration:

```xml
        <media folder="media" destination="plg_fields_plupload">
                <folder>css</folder>
                <folder>js</folder>
        </media>
```

The ZIP file has the folders `css` and `js`, and Joomla installs the extension.

But, JED Checker (develop branch) reports this error:

Rule XMLFILES - XML Files references
1 Errors - Click to View Details
#001 /plg_fields_plupload-main/plupload.xml
Folder not found: media/css
Folder not found: media/js

CC: @dryabov

**Comment by dryabov** [comment]
Unable to reproduce with the following plugin structure (and above code in the plugin's manifest):
```
├ plg_test.xml
├ test.php
└ media
  ├ css
  │ └ test.css
  └ js
    └ test.js
```
Are those `css` and `js` folders located in the `media` folder (as stated in the manifest)?

**Comment by anibalsanchez** [comment]
You can test it with the real extension: https://extensions.joomla.org/extension/plupload-for-joomla/

**Comment by dryabov** [comment]
I've checked. Joomla's `JInstaller::parseMedia` uses `media` attribute if only corresponding directory exists. Most likely in this case the JEDChecker should only warn on "media" directory doesn't exist. I'll prepare a patch later.
